domino.go

```diff
@@ -283,6 +317,17 @@ type KeyValue struct {
 /************************************** GetItem ****************************************/
 /***************************************************************************************/
 type get dynamodb.GetItemInput
+type output struct {
+	*dynamodb.GetItemOutput
+	Error error
+}
+
+func (o *output) Result(item interface{}) (err error) {
```

Comment:
This essentially allows us to deserialize lazily and still have access to the error and data object? 👍